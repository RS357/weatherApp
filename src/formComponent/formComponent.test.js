import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import FormComponent from './formComponent.js';

describe(('FormComponent should'), () => {
    it ('exist', () => {
        expect(FormComponent).toBeDefined()
    });

    it('should have a consistent structure', () => {
        const wrapper = mount(<FormComponent/>)
        expect(toJson(wrapper)).toMatchSnapshot()
    })

})
