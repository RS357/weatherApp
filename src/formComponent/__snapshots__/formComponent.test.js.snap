// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormComponent should should have a consistent structure 1`] = `
<FormComponent>
  <form>
    <label>
       Country:
      <Textbox
        city=""
        handleChange={[Function]}
      >
        <input
          onChange={[Function]}
          placeholder="city"
          type="text"
          value=""
        />
      </Textbox>
    </label>
    <SubmitComponent
      handleSubmit={[Function]}
    >
      <input
        onClick={[Function]}
        type="submit"
        value="submit"
      />
    </SubmitComponent>
  </form>
</FormComponent>
`;
